src/pd/client.rs

```diff
@@ -413,4 +413,24 @@ impl PdClient for RpcClient {
             .request(req, executor, LEADER_CHANGE_RETRY)
             .execute()
     }
+
+    fn scatter_region(&self, mut region: RegionInfo) -> Result<()> {
```

Comment:
only use in test?

Reply:
No, it will be used in the import service.